docs/reference/glossary.md

```diff
@@ -163,6 +167,10 @@ A message contains information to be delivered to interested parties during exec
 
 - [Messages](/components/concepts/messages.md)
 
+### Orchestration cluster
+
+An orchestration cluster includes Zeebe, Operate, Tasklist, and Optimize, existing alongside the management cluster.
```

Comment:
Orchestration cluster= Orchestration Cluster Core + Connectors+ Connectors Runtime
 Orchestration Cluster Core = Zeebe, Operate, Tasklist,  Optimize and identity

Reply:
I added both terms, but referred to the "Orchestration Cluster Core" as follows:

> ### Orchestration core
> An orchestration core or orchestration cluster core includes Zeebe, Operate, Tasklist, Optimize, and Identity.

Would this work @theburi? Or do you want use to be specific and refer to this as "orchestration cluster core"?